packages/dhis2/src/Adaptor.js

```diff
@@ -425,10 +475,10 @@ export function update(
 
 /**
  * Get data. Generic helper method for getting data of any kind from DHIS2.
- * - This can be used to get `DataValueSets`,`events`,`trackedEntityInstances`,`etc.`
+ * - This can be used to get `DataValueSets`,`events`,`trackers`,`etc.`
  * @public
  * @function
- * @param {string} resourceType - The type of resource to get(use its `plural` name). E.g. `dataElements`, `trackedEntityInstances`,`organisationUnits`, etc.
+ * @param {string} resourceType - The type of resource to get(use its `plural` name). E.g. `dataElements`, `tracker/trackedEntities`,`organisationUnits`, etc.
```

Comment:
I am tempted to rename this to `path` instead of `resourceType`. Maybe that's too bige a change. Maybe I'll canvas for opinions tomorrow

Reply:
Still thinking about this, but I need to hold the whole dhis2 API in my head and I can't do that at the moment :(